ied-be/src/templates/certificates/ied/certificate.css

```
@@ -0,0 +1,256 @@
+@page {
+  size: A4 landscape;
```

Stylelint (17.9.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)